src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -16,8 +24,28 @@
  * <p>Приклад 2: A _f*c a?F
  * Результат 2: acf</p>
  */
+
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    private static final int OUTPUT_LIMIT = 5;
+
+    public String getUniqueCharacters(String fileName) throws FileNotFoundException {
+        Set<Character> treeSet = new TreeSet<>();
+        String line = "";
+        try (BufferedReader input = new BufferedReader(new FileReader(fileName));) {
+            int character;
+            while ((character = input.read()) != -1) {
+                if (Character.isLetter(character)) {
+                    treeSet.add((char) Character.toLowerCase(character));
+                }
+            }
+        } catch (IOException e) {
+            throw new FileNotFoundException("File is not found " + e);
+        }
```

Comment:
let's just leave throws IOException here. You cannot wrap any IOException to FileNotFoundException